index.html

```diff
@@ -66,7 +66,12 @@
       <!-- Intro -->
       <section id="intro" class="wrapper style1 fullscreen fade-up">
         <div class="inner">
-          <h1>CODEVID-19</h1>
+          <div class="image">
+            <img
+              src="/images/codevid-19_whitelogo.png"
+              style="max-width: 75%; margin: 0 1.5em 1em 0; top: 0.25em"
```

Comment:
We should have alt text here. Also for SEO it's good to keep the `<h1>` as Google, et al rely on it for understanding the page. We can use the `<header>` tag to flag this for an image.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header

If you'd like to merge this PR I can update.

Reply:
Ah, glad there's a way to do this and I've learned something.  Will review your new PR soon™